sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -1,5 +1,5 @@
 ---
-parent: 'common/test_cases/origin.yml'
+parent: 'common/test_cases/origin_minimal.yml'
 overrides:
   junit_analysis: False
   timer: 'H H * * *'
```

Comment:
Is this timer still needed?

Reply:
Do we have it as a postsubmit in `openshift/release`?

Reply:
Yes, these jobs are all pre/post submits in their respective repositories.

Reply:
OK, then no timer needed here any longer

Reply:
Removed.